website/ref/index.toml

```diff
@@ -1,55 +1,47 @@
 [[articles]]
 name = "language"
-title = "The Language"
+title = "The Language Specification"
 
 [[articles]]
 name = "builtin"
-title = "The Builtin Module"
+title = "Builtin Functions and Variables"
 
 [[articles]]
 name = "edit"
-title = "The Editor Module (edit:)"
+title = "API for the Interactive Editor (edit:)"
```

Comment:
I see you reverted the change of putting the module names at the front; I prefer keeping that change, but I can do that myself too.